### PR TITLE
Implement Security and SeeAlso

### PR DESCRIPTION
**Pull request by yannanwang1**

### CODE REVIEW

File: Microsoft.RestApi/Models/Schema/FlowEntity.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace Microsoft.RestApi.Models
+{
+    using System.Collections.Generic;
+
+    using YamlDotNet.Serialization;
+
+    public class FlowEntity
+    {
+        [YamlMember(Alias = "name")]
```
Comment (partychen) [inline Microsoft.RestApi/Models/Schema/FlowEntity.cs]: Can this extend from NameEntity, and then no need to define Name property.

File: Microsoft.RestApi/Transformers/RestOperationTransformer.cs [hunk 2]
```diff
@@ -257,5 +257,119 @@ private static IList<DefinitionEntity> TransformDefinitions(OpenApiDocument open
...
+
+        public static IList<SeeAlsoEntity> TransformExternalDocs(OpenApiOperation openApiOperation)
+        {
+            var seeAlsoEntities = new List<SeeAlsoEntity>();
+            if (openApiOperation.ExternalDocs != null)
+            {
+                var seeAlsoEntity = new SeeAlsoEntity
+                {
+                    Description = openApiOperation.ExternalDocs.Description,
+                    Url = openApiOperation.ExternalDocs.Url.ToString()
```
Comment (partychen) [inline Microsoft.RestApi/Transformers/RestOperationTransformer.cs]: can this openApiOperation.ExternalDocs.Url be null? If yes, we could use openApiOperation.ExternalDocs.Url?.ToString()

Comment (yannanwang1) [reply Microsoft.RestApi/Transformers/RestOperationTransformer.cs]: Url field is required in the openApi spec.

Comment (partychen) [reply Microsoft.RestApi/Transformers/RestOperationTransformer.cs]: ok

**Pull request merged**